src/main/java/frc/robot/subsystems/arm/ArmConstants.java

```diff
@@ -0,0 +1,42 @@
+package frc.robot.subsystems.arm;
+
+public class ArmConstants { // TODO:
+
+    // Arm constants
+    public static final double CONFIG_VOLT_COMP = 10; //[voltage]
```

Comment:
```suggestion
    public static final double CONFIG_VOLT_COMP = 10; //[V]
```
Also:
1. The word "config" is redundant, it's obvious this constant is used for configuration.
2. Don't shorten words.